docs/tutorial-sokoban.rst

```diff
@@ -0,0 +1,401 @@
+Sokoban Tutorial
+****************
+
+When looking for simple games to implement on PewPew, it helps to look back to
+the times when computers were so young and underpowered, compared to today, that
+simple games were all they could run. One of those games is Sokoban. Let’s try
+to write a Sokoban for PewPew!
+
+Sokoban, which you can play in your browser at https://www.sokobanonline.com/,
```

Comment:
I'm not sure we want to advertise that website. Maybe we could link to https://en.wikipedia.org/wiki/Sokoban instead?

Reply:
Good point, I didn’t spend much thought on that. Fixed.